csaf_2.0/prose/csaf-v2-editor-draft.md

```diff
@@ -1573,29 +1992,32 @@ Recommended (`recommended`) of value type Products (`products_t`) represents tha
 
 Under investigation (`under_investigation`) of value type Products (`products_t`) represents that it is not known yet whether this version is or is not affected by the vulnerability. However, it is still under investigation - the result will be provided in a later release of the document.
 
-##### 3.2.3.1.9 Vulnerabilities Property - Vulnerability - References
+#### 3.2.3.9 Vulnerabilities Property - References
 
 References (`references`) have value type References (`references_t`).
```

Comment:
```suggestion
References (`references`) of value type References (`references_t`) holds a list of related external references.
```

Reply:
This might be misleading as we also allow the type `self`.

Reply:
This is now tracked in #223. The solution will be probably part of one of the next editor revisions.